cluster.go

```diff
@@ -218,7 +218,7 @@ type CNode interface {
 // latency funcs return true if alls well, false if they should be marked as failing
 func defaultLatencyFunc(c *Client) bool {
 	cmd := c.Ping(context.Background())
-	if isLoadingError(cmd.err) {
+	if cmd.err != nil && isLoadingError(cmd.err) {
```

Comment:
i would add the nil check in isLoadingError(cmd.err) as there could potentially be other places where the api would be called

Reply:
Every existing  place in the code that calls `is*Error` guards against nils one way or the other, this is staying consistent with the pattern established while fixing the problem in the simplest way possible.